integration-tests/pom.xml

```diff
@@ -38,26 +41,38 @@
             <version>${spring-web.version}</version>
```

Comment:
Let's get rid of the `spring-web` dependency. I only included it to have the RestTemplate and make http requests to the Neo4J server. I like the idea to test it using the `neo4j-java-driver`

Reply:
Agreed.